packages/runtime/src/parser/parser.ts

```diff
@@ -960,3 +960,30 @@ export function getProgramVersion(program: string): number {
 	opcodeFromSentence(wordsFromLine(firstLine), 1, ip, ExecutionMode.APPLICATION);
 	return ip.tealVersion;
 }
+
+/**
+ *
```

Comment:
lets add a description

Reply:
description? I have explained it in `@returns` part